app/main.py

```diff
@@ -1 +1,51 @@
-# write your code here
+from __future__ import annotations
+
+
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
+
```

Comment:
```suggestion
        self.skills += ["JavaScript", "HTML", "CSS"]

```
and the same for others